articles/install-guide/update.md

```diff
@@ -14,67 +14,102 @@ Learn how to update the Microsoft Quantum Development Kit (QDK) to the latest ve
+
+You can now skip ahead to [update your Visual Studio QDK extension](#update-visual-studio-qdk-extension).
+
+
+### Update Q# projects in Visual Studio Code
 
 1. In Visual Studio Code, open the folder containing the project to update
-1. Select Terminal > New Terminal
-1. Follow the instructions for updating using the command line
+2. Select **Terminal** -> **New Terminal**
+3. Follow the instructions for updating using the command line (directly below)
 
-### Using the command line
+### Update Q# projects using the command line
 
 1. Navigate to the folder containing your project file
 2. Run the following command:
-```bash
-dotnet clean [project_name].csproj
-```
-
-3. [Update the target framework](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks) in each of your .csproj files to netcoreapp3.0 (or netstandard2.1 if it is a library project)
+	```bash
+	dotnet clean [project_name].csproj
+	```
+
+3. In each of your .csproj files, update the target framework to `netcoreapp3.0` (or `netstandard2.1` if it is a library project).
+	That is, edit lines of the form:
+	```xml
+	<TargetFramework>netcoreapp3.0</TargetFramework>
+	```
+	You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
 4. Run the following command:
-```bash
-dotnet add package Microsoft.Quantum.Development.Kit
-```
-if your project uses any other Microsoft.Quantum packages, run the command for these too.
+	```bash
+	dotnet add package Microsoft.Quantum.Development.Kit
+	```
 
-5. Save and close all files
+	If your project uses any other Microsoft.Quantum packages, run the command for these too.
+5. Save and close all files.
 6. Repeat 1-4 for each project dependency, then navigate back to the folder containing your main project and run:
-```bash
-dotnet build [project_name].csproj
-```
+	```bash
+	dotnet build [project_name].csproj
+	```
+
+With your Q# projects now updated, follow the instructions below to update the QDK itself.
+
+## 2. Updating the QDK
+
+The process to update the QDK varies depending on your development language and environment.
+Select yours below to be taken to the proper instructions.
+
+* [Python: update the IQ# extension](#update-iq-for-python)
+* [Jupyter Notebooks: update the IQ# extension](#update-iq-for-jupyter-notebooks)
+* [Visual Studio: update the QDK extension](#update-visual-studio-qdk-extension)
+* [VS Code: update the QDK extension](#update-vs-code-qdk-extension)
+* [Command-line and C#: update project templates](#c-using-the-dotnet-command-line-tool)
+
 
-## Update IQ# for Python
+### Update IQ# for Python
 
 1. Update the `iqsharp` kernel
 
     ```bash
     dotnet tool update -g Microsoft.Quantum.IQSharp
     dotnet iqsharp install
     ```
+	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, do not fret. 
```

Comment:
```suggestion
	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, you can keep following the steps below and enter the command again after step 3. 
```

Reply:
Yeah I was unsure myself, just put the workaround as a patch solution. Upon now testing it again, I didn't get the error message, but I'm not sure if that means it's fixed for everyone